libs/design/src/molecules/paginator/paginator.component.html

```diff
@@ -0,0 +1,14 @@
+<span (click)="onNotifyPrevPageChange()" *ngIf="showPrevPageChange()" class="daff-paginator__prev daff-paginator__element">Prev</span>
+
+<span class="daff-paginator__elipsis" *ngIf="showFirstElipsis()"><span class="daff-paginator__element" (click)="onNotifyPageChange(1)" [class.daff-paginator__element--selected]="isSelected(1)">1</span>...</span>
+<span class="daff-paginator__element-wrapper" *ngFor="let pageNumber of _numberOfPagesArray">
```

Comment:
`daff-paginator__page-item` instead of `daff-paginator__element-wrapper`

Reply:
I just removed this class, because it wasn't needed